src/CLI/renderer/templates/image_show.j2

```diff
@@ -0,0 +1,13 @@
+{{'-----------------------------------------------------------'}}
+{% for key,value in json_output.items() %}
+{% if key == 'Available' %}
+{{key.ljust(20)}}:
```

Comment:
This file is no longer used right? Could you delete it?

Reply:
OK